types/mercadopago/shared/currency.enum.d.ts

```diff
@@ -0,0 +1,360 @@
+export enum Currency {
```

Comment:
In that case this needs to be type of string literal unions instead  and not enum since there is no value associated...

Reply:
Sorry but what is the actual difference? I mean, we could use literal unions but it would have the same effect as an enum. Check [this example](https://www.typescriptlang.org/play?#code/KYOwrgtgBAwmBO9QGMCeUDeAoKUD0AVAVAKogCWALsACZQCC8AhgEZQCiE5z1AzlDW4ALJtAJ4cDdgBEoAXigByejMUAaSYWL0AZgHMRIKE32HyUcZPoAxAHLylN24qwBfLFkqoADsFgIkEDQAJgdNIlIKajpGVg4uHmB+QXgRMQlcZVUoAB9w7VMmIxMDIvNLTKdFAG4PABtgSmMALn9EFHQFOHag1AA6FWlahqaWVu7AkIcs6RqPZAB7EF4Fhr66hb0ACkUANXoAGRalNWNTr18FnWMASixF5dXgdc2d-aOxk6gWc59gK++dweKzWG22iiY8jkChY6mMUJhNyAA) using enum and literal unions. I would rather enum because I can comment each value and when you hover it, it will give you a complete description like hovering `Currency.CAD` would give me `Canadian dollar`. I am ready to make the changes to use string literal unions, but I want to make sure it is really needed.

Reply:
There exists no Currency value so there cant be enum definition as enum defines that there is value Called Currency with properties you defined at run time.